src/config/index.jsx

```diff
@@ -0,0 +1,3 @@
+export default {
+    BASE_URL: process.env.REACT_APP_BASE_URL
+  };
```

Comment:
new line here

Reply:
> new line here

I have implemented the feedback @victorjambo